tests/end-to-end/harbor/use-api.bats

```diff
@@ -1,19 +1,22 @@
 #!/usr/bin/env bats
 
-# bats file_tags=harbor,use-api
-
 # End-to-end test: Harbor use API
 # Same as integration test without local-cluster setup and with verify tls.
 
 setup_file() {
   export BATS_NO_PARALLELIZE_WITHIN_FILE=true
   export CK8S_AUTO_APPROVE="true"
 
-  load "../bats.lib.bash"
+  load "../../bats.lib.bash"
   load_common "harbor.bash"
   load_common "yq.bash"
   load_assert
 
+  mark.setup
+  mark.punch
+
+  export TESTS_MARKER
```

Comment:
I assume this is to bail early if one of the test fails? Did just a quick search and found `$BATS_TEST_COMPLETED`, have you tried checking that in teardown and skip the rest of the run?